tests/trig.rs

```diff
@@ -153,46 +153,44 @@ impl SynthLanguage for Trig {
     fn get_lifting_rules() -> Ruleset<Self> {
         Ruleset::new(&[
             // definition of sine, cosine, tangent
+            // (sine)
             "(sin ?a) ==> (/ (- (cis ?a) (cis (~ ?a))) (* 2 I))",
             "(/ (- (cis ?a) (cis (~ ?a))) (* 2 I)) ==> (sin ?a)",
+            // (sine, alternatively)
```

Comment:
why do we need the alternative ones?

Reply:
Hey, I actually found an additional rule after removing these.